sources/Parser/ParserYaml.cpp

```diff
@@ -127,10 +127,14 @@ namespace BBM {
 		std::ofstream blockFile(block);
 		std::ofstream playerFile(player);
 		std::ofstream bonusFile(bonus);
+		auto &ret = scene->view<TagComponent<Timer>, TimerComponent>();
 
+		for (auto &[myEntity, tag, timerComponent] : ret) {
```

Comment:
You shouldn't do a for when you want to serialize only one entity. You can access the first entity of the view with a .front()

Reply:
fixed